ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly.yaml

```diff
@@ -807,6 +807,19 @@ tests:
     test:
     - ref: openshift-extended-test
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-winc
+- as: aws-ipi-ovn-winc-fips-f14
+  cron: 11 11 20 * *
```

Comment:
Pls follow https://github.com/openshift/release/blob/8bcf538fefce7bb223a322ee6c054d5e80f2a670/ci-operator/config/openshift/openshift-tests-private/README.md#job-frequency to set cron item.